packages/vendure-plugin-campaign-tracker/README.md

```
@@ -0,0 +1,67 @@
+# Vendure Campaign Tracker plugin
+
+### [Official documentation here](https://pinelab-plugins.com/plugin/vendure-plugin-campaign-tracker)
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)